src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/actions/manual/ManualTriggerAction/index.jelly

```diff
@@ -132,8 +132,19 @@
                                                                    id="check${theId}"
                                                                    onClick="rowSelected('${theId}')"/>
                                                         </td>
-                                                        <td onClick="activateRow('${theId}')">
```

Comment:
Initially, I removed it, because it activated row even if you clicked on the link to navigate to Gerrit.
I added the functionality back in following commit and now click event is consumed after clicking on the link, so it no longer happens.